.github/workflows/build.yml

```
@@ -19,4 +19,96 @@
+      run: dotnet build TestStream.Runner/TestStream.Runner.csproj
+    
+    - name: Trigger Azure DevOps Pipeline
+      env:
+        AZURE_DEVOPS_ORG: nanoframework
+        AZURE_DEVOPS_PROJECT: nanoFramework.IoT.TestStream
+        AZURE_DEVOPS_PIPELINE_ID: 111
+        AZURE_DEVOPS_PAT: ${{ secrets.AZURE_DEVOPS_PAT }}
+        AZURE_POOL_NAME: TestStream
+      run: |
+        pwsh -Command "
+        
```

yamllint

[error] trailing spaces

(trailing-spaces)